sensor_msgs/Image image_raw
sensor_msgs/Image image_undistorted